content/news/buzz/2025-curtin-capstone-semester-end-blog/contents.lr

```diff
@@ -0,0 +1,102 @@
+ - Caydn Lee
+	 - Discipline: Software Engineering
+	 - GitHub: [caydnn](https://github.com/caydnn)
+ - Jaeden Mah
+	 - Discipline: Computer Science
+	 - GitHub: [JMah007](https://github.com/JMah007)
+ - Mitchell Pontague
+	 - Discipline: Software Engineering
+	 - GitHub: [mEp3ii2](https://github.com/mEp3ii2)
+ - Veronica Taniputra
+	 - Discipline: Cyber Security
+	 - GitHub: [vt37](https://github.com/vt37)
+
+## What We’ve Worked On
+
+Throughout the first semester, our team has been focused on contributing to issues on Briefcase and Toga while working on our research to prepare for what is to come in the following semester. We’ve also worked on implementing several of the Toga Web widgets to enhance functionality on the web backend before working on the web test suite for next semester.
+  
+We were first tasked with contributing to a first timer issue before moving on to other more advanced issues.
+
+### Briefcase Contributions
+
+- PR [#2198](https://github.com/beeware/briefcase/pull/2198): Add boolean question (feature)
+- PR [#2103](https://github.com/beeware/briefcase/pull/2203): Add XML content escaping filter to cookiecutter.py (feature)
+- PR [#2199](https://github.com/beeware/briefcase/pull/2199): Add catch exception when deleting JDK (bugfix)
+- PR [#2229](https://github.com/beeware/briefcase/pull/2229): Accept other changelog name and extension (feature)
+- PR [#2201](https://github.com/beeware/briefcase/pull/2201): Add --app option to briefcase build/package (feature)
+- PR [#2214](https://github.com/beeware/briefcase/pull/2214): Add --app option to briefcase create/update (feature)
+- PR [#2236](https://github.com/beeware/briefcase/pull/2236): Normalise contribution docs with Toga (documentation)
+- PR [#54](https://github.com/beeware/briefcase-windows-VisualStudio-template/pull/54): Use XML content escaping filter to Visual Studio template. (bugfix) (work in progress)
+
+### Toga Contributions
+
+- PR [#3259](https://github.com/beeware/toga/pull/3259): Add web screenshots (documentation)
+- PR [#3466](https://github.com/beeware/toga/pull/3466): Fix button click handling in Toga Web backend to correctly trigger event (bugfix)
+
+### Web Widgets
+
+- PR [#3338](https://github.com/beeware/toga/pull/3338): Date Input
+- PR [#3405](https://github.com/beeware/toga/pull/3405): TimeInput
+- PR [#3362](https://github.com/beeware/toga/pull/3362): ScrollContainer
+- PR [#3425](https://github.com/beeware/toga/pull/3425): Table (work in progress)
+- PR [#3402](https://github.com/beeware/toga/pull/3402): Selection
+- PR [#3527](https://github.com/beeware/toga/pull/3527): Slider
+
+## What We're Working Towards
+
+Now that we’re more familiar with the BeeWare ecosystem, through both our initial PRs and widget implementations, and having concluded our semester 1 research phase, we’re shifting our focus to larger, more structural improvements for the second half of the project.
+
+Our primary goals moving forward include:
+
+### Toga Web Testing
```

Comment:
This is all good content, but possibly a little verbose for this specific venue - 2-3 paragraphs would be ideal. We don't need to dig into the specifics of the implementation; a vague hint in the direction of "a mechanism for proxying widget requests into the browser, and Playwright to check browser state" is enough, with a link off to the discussion post for more details.